docs/api-and-integration/metadata/index.md

```diff
@@ -0,0 +1,116 @@
+
+To create the metadata file and get it to Octopus, use the Octopus _Metadata_ step in your build server (below is the TeamCity step for illustration).
+
+![Package Metadata Step](metadata-step.png)
+
+The build server plugins work on the assumption that the development team references the work-items in their commit messages, for example, as they would for Bamboo build integration with Jira or integration with GitHub issues.
+
+The metadata will appear in the package feed details for any package in the internal feed that has had metadata pushed for it.
+
+![Package Details](package-detail.png)
+
+If there is a specific issue tracker extension, e.g. Jira, enabled and configured that matches the work-items type the work-items will appear as a link to the issue tracking system. If not, the work-items will appear as plain text.
+
+## Project Settings
+
+The next step is to tell your Octopus project which packages in its deployment process contain metadata that you want included in the releases and deployments.
+
+![Octopus Project work-item settings](octo-project.png)
+
+In this example, the project is using the primary package from the step named Deploy. You can specify multiple packages if required. An example of where you might use this is a deployment process that includes steps for a Web App and a SQL database schema migration. The packages for both of those form the complete product and either or both could be contributing work-items.
+
+## Releases and Deployments
+
+When a release is created for the project, the selected package versions are used to collect the work-item list and it is shown with the release details.
+
+![Release work-items](release-work-items.png)
+
+When this release is deployed the metadata will be contributed to the deployment. They will appear in the preview.
+
+![Deployment preview](deploy-preview-work-items.png)
+
+They will also appear on the task summary for the deployment.
+
+![Deployment work-items](deploy-work-items.png)
+
+In some scenarios this one to one between releases and deployments will be the norm. Teams using continuous deployment may see a rapid progression of versions through their environments and on to production.
+
+Many teams won't be operating like this though, many will accumulate a number of releases before promoting to the next environment. For example, they may merge pull requests for several fixes/features and then deploy the final release to test. This could even go through a couple of revolutions and then eventually a final release progresses from test to production. In this situation the deployment to production isn't just the work-items from that last release, it is the accumulation of all of the work-items in all of the releases since the last one that was deployed to production.
```

Comment:
```suggestion
Many teams won't be operating like this though. It is common that a number of releases will be prepared and tested before promoting to the next environment. For example, they may merge pull requests for several fixes/features and then deploy the final release to test. This could even be repeated several times and then eventually a final release progresses from test to production. In this situation the deployment to production isn't just the work-items from that last release, it is _the accumulation of all of the work-items in all of the releases since the last one that was deployed to production_.
```